### PR TITLE
Bulk add markers

### PR DESCRIPTION
**Pull request by cicloon**
The title says it all. Previously the plugin only had interfaces for adding and removing markers one by one. I added those, plus the refactoring needed to keep the code DRY.

### CODE REVIEW

**Comment by larahoya** [comment]
I've only reviewed the JS part of this PR. 👍 👍 👍

File: src/android/plugin/google/maps/PluginMarker.java [hunk 1]
```diff
@@ -58,18 +61,29 @@
...
+
+
   @SuppressWarnings("unused")
-  private void createMarker(final JSONArray args, final CallbackContext callbackContext) throws JSONException {
+  private PluginAsyncInterface noOpAsyncInterface() {
+    return new PluginAsyncInterface() {
+
+      @Override
+      public void onPostExecute(Object object){
+        Log.w("client", "noOpAsyncInterface onPostExecute");
```
Comment (pedroduran) [inline src/android/plugin/google/maps/PluginMarker.java]: Do we really print a warning when this happens?

**Review by pedroduran — APPROVED** [review]
Besides the warning logs (I'd remove almost all them or at least reduce their log level) the code looks better than which we had and if works as should to, it's good enough for me!!

File: src/android/plugin/google/maps/PluginMarker.java (hunk 1 above)
Comment (cicloon) [reply src/android/plugin/google/maps/PluginMarker.java]: No, not really 😅

**Pull request merged**